src/overflow/setup_bands.py

```diff
@@ -0,0 +1,32 @@
+from osgeo import gdal
+from contextlib import contextmanager
+
+
+gdal.UseExceptions()
+
+
+@contextmanager
+def setup_bands(input_path, output_path):
+    n_bands = 1
+    input_raster = gdal.Open(input_path)
+    driver = gdal.GetDriverByName("GTiff")
+    output_dataset = driver.Create(
+        output_path,
+        input_raster.RasterXSize,
+        input_raster.RasterYSize,
+        n_bands,
+        gdal.GDT_Float32,
```

Comment:
Output dataset might not by float32, maybe we can default to the same as the input and add an optional parameter to set a different type?